pandas/core/series.py

```diff
@@ -5745,7 +5745,50 @@ def to_period(self, freq: str | None = None, copy: bool | None = None) -> Series
     _info_axis_name: Literal["index"] = "index"
 
     index = properties.AxisProperty(
-        axis=0, doc="The index (axis labels) of the Series."
+        axis=0, doc="""The index (axis labels) of the Series.
```

Comment:
https://github.com/pandas-dev/pandas/actions/runs/4584805993/jobs/8096531909?pr=51842#step:4:274:

Instead of putting `axis` and `doc` on the same line, you should use two lines:

```suggestion
        axis=0, 
        doc="""The index (axis labels) of the Series.
```

Reply:
https://github.com/pandas-dev/pandas/actions/runs/4584805993/jobs/8096531828?pr=51842#step:8:12

The first line should be in the line after the opening quotes:

```suggestion
        axis=0, 
        doc="""
        The index (axis labels) of the Series.
```